src/pages/Trade/Trade.tsx

```diff
@@ -88,6 +88,15 @@ export const Trade = memo(() => {
     return translate('navBar.trade')
   }, [location.pathname, translate])
 
+  // Only rewrite for /trade (input) else problems, we'll be redirected back to input on confirm
```

Comment:
@NeOMakinG FYI, would be nice to rewrite *any* `trade path with pair and amount bits, not just the main (input) one, so users could e.g reopen a closed tab where they had confirm ready, and be routed back to input where they could take over that trade again, but definitely a story for another day!